### DISCUSSION TITLE
MARKER_MONITORING for Optimization

### DISCUSSION THREAD

**Discussion by ardao13** [su2code/SU2#2189] [Q&A]
Hello SU2 community,

I'm trying to do multi-objective optimization using discrete adjoint and FFD boxes for an aircraft geometry. I am planning to define two objectives as follows:

1.  DRAG (of whole aircraft)
2.  MOMENT_Y (only for wings)

Can I do that with shape_optimization.py? 

I could not find a way since I can only define **MARKER_MONITORING** once. I am open to your suggestions for dealing with this situation.

**Comment by bigfooted** [comment]
Maybe better to switch to the FADO method, it is more flexible, the tutorial is here: https://su2code.github.io/tutorials/Species_Transport/
You can then run 2 adjoint computations with it, one for drag and one for the moment-y and combine the information from the 2 runs. 
I am not sure what the easiest setup would be, there are several options here. Maybe you can start with a setup with 1 SU2 mesh with 1 FFD box for the wings and just first optimize that based on drag and then moment-y. This is probably the easiest to set up.
Then you could add to that a second adjoint run for the drag on the rest of the aircraft, with a second FFD box. But there might be some issues on the intersection between the fuselage and the wings so you might want to constrain movement there.
